terraform/a2ha-terraform/reference_architectures/existing_nodes/main.tf

```diff
@@ -223,6 +228,24 @@ module "bootstrap_automate" {
   chef_server_custom_certs_enabled = var.chef_server_custom_certs_enabled
   postgresql_custom_certs_enabled  = var.postgresql_custom_certs_enabled
   opensearch_custom_certs_enabled  = var.opensearch_custom_certs_enabled
+  opensearch_root_cert               = var.opensearch_root_cert
+  postgresql_root_cert               = var.postgresql_root_cert
+  managed_opensearch_domain_name     = var.managed_opensearch_domain_name
+  managed_opensearch_certificate     = var.managed_opensearch_certificate
+  managed_opensearch_domain_url      = var.managed_opensearch_domain_url
+  managed_opensearch_user_password   = var.managed_opensearch_user_password
+  managed_opensearch_username        = var.managed_opensearch_username
+  aws_os_snapshot_role_arn           = var.aws_os_snapshot_role_arn
+  os_snapshot_user_access_key_id     = var.os_snapshot_user_access_key_id
+  os_snapshot_user_access_key_secret = var.os_snapshot_user_access_key_secret
+  managed_rds_instance_url           = var.managed_rds_instance_url
+  managed_rds_superuser_username     = var.managed_rds_superuser_username
+  managed_rds_superuser_password     = var.managed_rds_superuser_password
+  managed_rds_dbuser_username        = var.managed_rds_dbuser_username
+  managed_rds_dbuser_password        = var.managed_rds_dbuser_password
+  managed_rds_certificate            = var.managed_rds_certificate
+  setup_managed_services             = var.setup_managed_services
+  setup_self_managed_services        = var.setup_self_managed_services
```

Comment:
run `terraform fmt <path/to/file>` for this file and other .tf files to format  all indentation